diesel_compile_tests/Cargo.lock

```diff
@@ -0,0 +1,678 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I think this is introduced unintentionally.

Reply:
I've added this lock file because otherwise some dependencies won't build with the old nightly version used by compile_test. The alternative would be to update specific crates manually as done for the MSRV ci run. As the compile-test live in their own workspace that should be fine.

Reply:
Ah, I see! Then it should be fine.